src/pages/resultsView/coExpression/CoExpressionViz.tsx

```diff
@@ -160,31 +162,39 @@ export default class CoExpressionViz extends React.Component<ICoExpressionVizPro
 
     private getPlotDataPromises(yAxisCoExpression?:CoExpressionWithQ) {
         const ret:{
-            molecularX:MobxPromise<NumericGeneMolecularData[]>,
-            molecularY:MobxPromise<NumericGeneMolecularData[]>|undefined,
+            molecularX:MobxPromise<NumericGeneMolecularData[]|GenesetMolecularData[]>,
```

Comment:
as far as I understand, the type should really be `MobxPromise<NumericGeneMolecularData[]> | MobxPromise<GenesetMolecularData[]>`. and for molecularY too